packages/flutter/lib/src/foundation/memory_allocations.dart

```diff
@@ -64,6 +64,8 @@ class ObjectCreated extends ObjectEvent {
   });
 
   /// Name of the instrumented library.
+  ///
+  /// Format chould be: 'package:flutter/rendering.dart'.
```

Comment:
```suggestion
  /// The format of this parameter should be a package Uri.
  ///
  /// For example: `'package:flutter/rendering.dart'` 
```
Another thought: Are we better off using a Uri, that validates its using a `package` scheme, instead of a string? (using an assert) Or is that done internally?

Reply:
It is more convention than rule. This is very low level class, that should not be concerned about complicated things and should work very-very fast, as flow of objects can be massive.